.github/workflows/aws-eks-verify.yml

```diff
@@ -74,8 +74,12 @@ jobs:
         id: probr
         uses: addnab/docker-run-action@v3
         with:
-          image: eknight/probr:v0.1.1 # https://github.com/probr/probr-docker
-          options: -v ${{ github.workspace }}/${{ env.PROBR_DIRECTORY }}:/probr/run
+          image: eknight/probr:v0.1.2-rc # https://github.com/probr/probr-docker
```

Comment:
Is there a reason to use:

eknight/probr:v0.1.2-rc

?

Reply:
Thats the version eddie has been working on that adds the ability to pass in credentials on the commandline.

Reply:
Let's move forward this way for now. We need a better/refined approach.